docs/source/reference/rest.md

```diff
@@ -4,15 +4,19 @@
 
 This section will give you information on:
 
-- what you can do with the API
-- create an API token
-- add API tokens to the config files
-- make an API request programmatically using the requests library
-- learn more about JupyterHub's API
+- [what you can do with the API](#what-you-can-do-with-the-api)
```

Comment:
I don't think in-page links like this are beneficial. There's already a table of contents links on the side bar that will always be correct, whereas these links will all break without warning if we change the spelling of headings.